docs/CustomUIA.md

```diff
@@ -0,0 +1,125 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT Licence. -->
+
+## Custom UI Automation properties
+Axe Windows supports the inspection of [custom UI Automation properties](https://docs.microsoft.com/en-gb/windows/win32/winauto/uiauto-custompropertieseventscontrolpatterns). The custom properties for which Axe Windows should register are described in a configuration file that is loaded once per session and applies to all inspected UI Automation elements.
+
+### Configuration file format
+Custom UI Automation configuration data is specified in a [JSON](https://en.wikipedia.org/wiki/JSON#Syntax) formatted text file in the configuration directory. The configuration file consists of one object with the following possible attributes:
```

Comment:
Suggestion: Omit the file location. This is another artifact of the initial planned implementation in AIWin, but there's no need for Axe.Windows to have this constraint. In fact, if/when we enable this in the CLI, we'll probably want to allow the user to specify the exact path to the config file to use.